\documentclass[a4paper,11pt]{article}
\usepackage[osf]{mathpazo}
\usepackage{ms}
\usepackage[]{natbib}
\usepackage{booktabs}
\raggedright

\newcommand{\smurl}[1]{{\footnotesize\url{#1}}}
\usepackage{graphicx}

\title{What we don't currently know about global plant functional diversity}
\author{
Will Cornwell$^1$
Will Pearse$^2$
Amy Zanne$^3$}

\affiliation{
*final list and order undecided\\
$^1$ University of NSW\\
$^2$ somewhere between Wales and Canada?\\
$^3$ global vagabond}
\date{}

\bibliographystyle{mee}

\usepackage[title,titletoc,toc]{appendix}

\mstype{Research Article}
\runninghead{What we don't know about plant diversity}
\keywords{}

\begin{document}
\mstitlepage
\noindent
% \doublespacing
% \linenumbers

\section{Summary}


\section{Introduction}

\section{Methods}

\section{Results and Discussion}

\subsection{Species  that we know a lot about}

Among the currently accepted 350,699 plant species in the world, there are about 1,647 that we know a lot about, spatially, genetically, and functionally--this is 0.47\% of extant diversity.  With more effort on taxonomy and synonymy that number may be a bit higher with existing data, but largely the vast majority of the missing species are  data that is yet to be collected.  There are however hot spots of knowledge: Ginkgoaceae and Welwitschiaceae are well characterised botanical oddities.  Among the families with more than 100 species, the largest of the gymnosperm families are the best characterised--Pinacae (45 out of 255 species) and Cupressaceae (15 out of 163). Betulaceae (19 out of 234) and Fagaceae (34 out of 1101) are also notable families with disproportionately high sampling.  

These 1,647 species tend to be very well sampled in GBIF, so the sampling per GBIF observation in the other databases tends be higher.  These include the dominant trees and shrubs of North America, Europe, and SE Australia.  For example the Diaz et al. analysis included more than half of the GBIF observations at 50 degrees N latitude, but less than 10 \% of the species at that latitude (Figure \ref{fig:zanne_diaz_gam}).  There is a set of species that is common in both GBIF and TRY.  

\subsection{Species that we know a little about}

For about 12 \% of global plant species---39,650 species (see Table \ref{tab:two_and_three_way_comparisons}), we know \emph{something} about their genetic make--up, \emph{something} their traits, and at least have a geo-referenced record for a location.  These are primarily temperate species (see figure \ref{fig:genbank_try_gam}).

Beyond these 39,650 species, the databases differ greatly in their coverage.  GBIF, which has by far the most complete sampling of the databases, is nonetheless extremely poorly sampled for mosses and liverworts---all of the top ten most undersampled families are either mosses or liverworts (See Figure \ref{fig:gbif_treemap} and Table \ref{undersampled_families}).  To a lesser extent mosses and liverworts are also undersampled for both genes and traits (Table \ref{undersampled_families}).  

For both traits and genes, Orchidaceae, despite its widespread cultivation and intense horticultural interest, remains undersampled (Table \ref{undersampled_families})---this is like both on account of its diversity---27732 accepted species---and the rarity and inaccessibility of many of them.  Acanthaceae is another family that is under-sampled for both genes and traits.  Myrtaceae was a surprising entry on the genetically undersampled list.  The family is well sampled within Australia but very poorly sampled across the rest of its distribution (Figure \ref{fig:hyp}).  

In Genbank we found data for 14,565 genera before scrubbing and 12,154 after \citep[see][for a detailed analysis of number of genes and phylogenetic distribution of the data]{hinchliff2014some}.  This is 71.4 \% of the genera that are currently accepted by the plant list.  TRY has data for 9,717 accepted genera or 66.7 \% of the accepted genera.  The glass--half empty view is that we know much more than we did in the past.  The glass--half empty view is that for one--third of world's plant \emph{genera} we know nothing about how they ecologically make their living.  

\subsection{Species that we know almost nothing about}

There are three classes of species that are very hard to say anything about.  First, are the species that have yet to be described.  Only true specialists can have a sense where in the world they are, whether they've already been collected and mis-identified, and how to move taxonomic knowledge forward.  Second, are the species that have accepted names, but no additional data.  Presumably there is a type specimen somewhere in the world, with a location, but that data has not been digitized or for some other reason .  There are efforts underway to try to address this problem (REF HERBARIA DIGITIZATION PROJECTS), and data for this issue will likely improve quickly.  In this area it is worth prioritizing herbaria that are known to have collections of the absent species.  

For species with no genetic data, bar coding projects \citep{li2015plant} will help, and as pointed out by \cite{hinchliff2014some}, if the goal is phylogenetic inference, there is a power sharing feature of nodes that are down the tree, as genetic data across many tips informs the inferred phylogeny. 

For species with no functional trait data, the current data is very highly focused on clades with economic importance including Poaceae (67\% sampled), Pinaceae (85\%), Sapotaceae (51 \%), and others.  There are also clades that are particularly unusual including Proteaceae \citep[73\% see][]{Cornwell}, Zamiaceae (67\%), and Lycopodiaceae (.  

\subsection{Data imputation and its connection to the comparative methods}

Data interpolation techniques rely principally on phylogenetic relatedness.  
Most, models of the expected similarity based on relatedness rely on an underlying implicit or explicit model of trait evolution.  
Usefully some of the recently proposed methods do allow for characterising uncertainty, this uncertainty in the typical case
does not include model uncertainty.  And as most simple models of trait evolution for plants are far from adequate \citep{pennell2014model}, and heterogeneity in evolutionary process among the branches of the tree is common \citep{eastman2011novel},
we have a ways to go before we fully can model the uncertainty in trait imputation approaches.   

The most difficult problem for data imputation is that for poorly sampled clades---we do not know enough to know if the evolutionary process is different in those clades.  As such there is no way to characterize model uncertainty for these groups until there is enough data to understand something about the evolutionary process for the traits of interest.  This problem may not be very important if the quantity of interest is a summary statistic, i.e. how many of the world's species are woody, but for goals that are specific to the unsampled species---what is the species' ecology and it's conservation risk---imputation will not be informative, given the current patchy data.  In other words, there is no magic substitute for real data.  

\subsection{Paths forward}



\section{Acknowledgement}

This research includes computations using the Linux computational cluster Katana supported by the Faculty of Science and the Center for Ecosystem Science, UNSW Australia.
This study would not have been possible without the 128GB of RAM available to each node on that cluster.  


\section{Tables}



\input{tables/all_families_ranking}

\input{tables/summary_of_endemic_analysis}

\input{tables/two_and_three_way_comparisons}

\section{Figures}

\begin{figure}[h!]
\centering
  \includegraphics[width=\textwidth]{figures/multi_gam.png}
   \label{fig:genbank_try_gam}
\end{figure}

\begin{figure}[h!]
\centering
  \includegraphics[width=\textwidth]{figures/treemap_well_known.pdf}
  \caption{Size of boxes represents the total species richness of each family (TPL 1.1) and the color represents the proportion of species which have been 
  well characterised both for functional traits and for genes.}
\end{figure}

\begin{figure}[h!]
\centering
  \includegraphics[width=\textwidth]{figures/multi_gam_zae_diaz.png}
  \label{fig:zanne_diaz_gam}
\end{figure}


\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{figures/treemap_try-database.pdf}
\caption{Size of the boxes are proportional to the species richness of each family.  Proportion of TPL 1.1 accepted names that were in TRY-db as of 2015.  }
\label{fig:try_treemap}
\end{figure}
\clearpage

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{figures/treemap_gbif.pdf}
\caption{Size of the boxes are proportional to the species richness of each family.  Proportion of TPL 1.1 accepted names that were in GBIF as of 2015.  }
\label{fig:gbif_treemap}
\end{figure}
\clearpage

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{figures/treemap_genbank.pdf}
\caption{Size of the boxes are proportional to the species richness of each family.  Proportion of TPL 1.1 accepted names that were in GBIF as of 2015.  }
\label{fig:genbank_treemap}
\end{figure}
\clearpage

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{figures/hyp.pdf}
\caption{ space by taxa.  }
\label{fig:hyp}
\end{figure}
\clearpage

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{figures/treemap_not_known.pdf}
\caption{The sampling of species by family for \emph{either} genetic or functional data.  Size of the rectangles represents the species richness of the group. }
\label{fig:hyp}
\end{figure}
\clearpage

\bibliography{refs}

\end{document}

